src/index.html

```diff
@@ -8,6 +8,78 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars__container">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+
```

Comment:
Can be without an empty line between elements whose tags consist of one line in 1 block.